app/api/tickets.py

```diff
@@ -248,6 +255,23 @@ def before_update_object(self, ticket, data, view_kwargs):
                     "Event having paid ticket must have a payment method",
                 )
 
+        if ticket.type == 'donation':
```

Comment:
There is an operator called `or`. Remove all the duplicated code and merge with paid ticket

Reply:
What about Error message?

Reply:
should i change it to `Select a payment method`??

Reply:
Donation is a paid ticket, Except the name and its function.

Reply:
> should i change it to `Select a payment method`??

Error message is fine, Also once you will make FE changes chances are less user will submit the form before completing the validations

Reply:
Done